packages/tracing/test/httpheaders.test.ts

```diff
@@ -227,7 +225,7 @@ describe('tracestate', () => {
       });
 
       it("won't include data set after first call to `getTraceHeaders`", () => {
-        expect.assertions(2);
+        expect.assertions(1);
```

Comment:
There are no need to explicitly state number of assertions in sync tests, but it's fine to leave it. Personal preference.

Reply:
^^^ Not from this PR (I hadn't rebased yet) but I did it because the expects are in a callback. Less of an issue in this test than some of the others, because the callback is explicitly called and included in the test itself, unlike, for instance, this one:

```
        it('uses existing sentry tracestate in envelope headers rather than creating a new one', () => {
          // one here, and two inside the `captureEvent` implementation above
          expect.assertions(3);
```